docs/podman-container-cleanup.1.md

```diff
@@ -8,7 +8,7 @@ podman\-container\-cleanup - Cleanup Container storage and networks
 
 ## DESCRIPTION
 **podman container cleanup** cleans up exited containers by removing all mountpoints and network configuration from the host.  The container name or ID can be used.  The cleanup command does not remove the containers.  Running containers will not be cleaned up.
-Sometimes container's mount points and network stacks can remain if the podman command was killed or the container ran in daemon mode.  This command is automatically executed when you run containers in daemon mode by the conmon process when the container exits.
+Sometimes container's mount points and network stacks can remain if the podman command was killed or the container ran in daemon mode.  This command is automatically executed when you run containers in daemon mode by the common process when the container exits.
```

Comment:
conmon is actually correct here.  Although it would probably be good to add a link to https://github.com/containers/conmon#conmon.   @mheon WDYT?

Reply:
Oh, it seems that I was to greedy in typo hunt. I'll change it back and we can add the link in another commit.